### DISCUSSION TITLE
Intel ARC OpenVino Windows without WSL

### DISCUSSION THREAD

**Discussion by BrowserOfThings** [vladmandic/sdnext#2039] [General]
Can i use SD.Next direct at Windows OS without WSL?

**Comment by Disty0** [comment]
Create a new install and use `webui.bat --use-openvino`

IPEX and WSL is still better for Intel ARC if you don't want to cap out at 768x768 on SD 1.5 or 1024x1536 on SDXL.
OpenVINO will need 16 GB RAM to compile SD 1.5 models and 48 GB RAM to compile SDXL models.
It will use the cached model after the first compile but changing the resolution or using Lora will recompile again.
